### ISSUE TITLE
ArticulationPoints duplicate vertices

### ISSUE THREAD

**Issue by le27** [gap-packages/Digraphs#140]
The articulation points function can return repeats of the same vertex. In the following example the vertex 30 is returned twice and the vertex 61 is returned 3 times:

~~~~
gap> gr:=DigraphFromSparse6String(
> ":~?@V`OINBg_McouHAxQD@gyYEW}Q_@_YdgE`?OgZgpEbfYQKDGqiDQEI`wGdjoADGZG\
FIJONFQSplq]y@IwvbPKhMh}JGK?OLzW{agKKfRCtarqTGayQGb]rMIurapkxPG?RGcI]\
IBtB_`EQKJ@LmxlL_?k^QieOkB|T");
<digraph with 87 vertices, 214 edges>
gap> ArticulationPoints(gr);
[ 30, 27, 30, 23, 21, 63, 12, 73, 52, 75, 84, 11, 17, 60, 19, 87, 15, 69, 76, 42, 61, 59, 61, 66, 61, 68, 8, 51, 41, 37, 36, 18, 79, 3, 
  46, 1 ]
gap> SortedList(ArticulationPoints(gr));
[ 1, 3, 8, 11, 12, 15, 17, 18, 19, 21, 23, 27, 30, 30, 36, 37, 41, 42, 46, 51, 52, 59, 60, 61, 61, 61, 63, 66, 68, 69, 73, 75, 76, 79, 
  84, 87 ]
~~~~

**Comment by james-d-mitchell** [comment]
I can reproduce this, and will investigate.

**Comment by james-d-mitchell** [comment]
It looks like all the vertices returned are articulation points at least.

**Comment by wilfwilson** [comment]
Resolved in v0.14.0 release
